scripts/push-and-review.sh

```
@@ -23,9 +53,23 @@ echo "=== PR #$PR_NUMBER found. Starting Claude review... ==="
 echo "CodeRabbit and Qodo will also review automatically."
 echo ""
 
-# Run the pokemon-reviewer subagent in the background
-# Uses Max plan (no API cost)
-claude --agent pokemon-reviewer --print "Review PR #$PR_NUMBER on branch $BRANCH. Run gh pr diff to see changes and post your review." &
+# Set up logging
+mkdir -p "$LOG_DIR"
+LOG_FILE="$LOG_DIR/review-PR${PR_NUMBER}-$(date +%Y%m%d-%H%M%S).log"
+
+# Clean up old logs (keep last 10)
+ls -t "$LOG_DIR"/review-*.log 2>/dev/null | tail -n +11 | xargs rm -f 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)